lib/web/apiserver.go

```diff
@@ -629,7 +629,7 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*APIHandler, error) {
 				// part[0] is empty space from leading slash "/"
 				// part[1] is the prefix "v1"
 				switch pathParts[2] {
-				case "webapi", "enterprise", "scripts", ".well-known", "workload-identity":
+				case "webapi", "enterprise", "scripts", ".well-known", "workload-identity", "web":
```

Comment:
are we sure we want to include every request that is prefixed with web or only web/config.js?

Reply:
i think it's fine because we are just saying if `/v1/web` strip the `/v1`